homeassistant/components/lifx/themes.py

```diff
@@ -0,0 +1,184 @@
+        (31, 1.0, 1.0, 3500),
+        (32, 1.0, 0.6, 2800),
+        (32, 1.0, 1.0, 4200),
+        (33, 1.0, 0.6, 4900),
+        (33, 1.0, 1.0, 2450),
+        (34, 1.0, 0.7, 2800),
+    ],
+    "hanukkah": [
+        (213, 0.52, 1.0, 3500),
+        (219, 1.0, 1.0, 3500),
+        (0, 0.0, 0.32, 7000),
+        (199, 1.0, 0.34, 3500),
+        (232, 1.0, 0.35, 3500),
+        (225, 0.25, 0.13, 3500),
+    ],
+    "holly": [
+        (117, 1.0, 1.0, 3500),
+        (116, 0.9, 1.0, 3500),
+        (1, 1.0, 1.0, 3500),
+        (118, 1.0, 0.5, 3500),
+        (360, 1.0, 0.9, 3500),
+    ],
+    "independence day": [
+        (360, 0.0, 1.0, 3500),
+        (360, 1.0, 1.0, 3500),
+        (240, 1.0, 1.0, 3500),
+    ],
+    "intense": [
+        (242, 0.75, 1.0, 3500),
+        (300, 1.0, 0.87, 3500),
+        (164, 0.99, 1.0, 3500),
+        (248, 1.0, 0.23, 3500),
+    ],
+    "mellow": [
+        (359, 0.31, 0.59, 3500),
+        (315, 0.24, 0.82, 3500),
+        (241, 1.0, 0.4, 3500),
+        (256, 0.36, 0.5, 3500),
+        (79, 0.05, 0.4, 3500),
+    ],
+    "peaceful": [
+        (198, 0.48, 0.11, 3500),
+        (2, 0.46, 0.85, 3500),
+        (54, 0.36, 0.85, 3500),
+        (4, 0.63, 0.56, 3500),
+        (203, 0.34, 0.56, 3500),
+    ],
+    "powerful": [
+        (10, 0.99, 0.66, 3500),
+        (59, 0.7, 0.98, 3500),
+        (11, 0.99, 0.41, 3500),
+        (61, 0.44, 0.99, 3500),
+        (18, 0.98, 0.98, 3500),
+        (52, 0.88, 0.97, 3500),
+        (52, 0.88, 0.97, 3500),
+    ],
+    "relaxing": [
+        (110, 0.95, 1.0, 3500),
+        (71, 1.0, 1.0, 3500),
+        (123, 0.85, 0.33, 3500),
+        (120, 0.5, 0.1, 3500),
+    ],
+    "santa": [
+        (0, 1.0, 1.0, 3500),
+        (351, 0.05, 1.0, 3500),
+        (2, 1.0, 0.58, 3500),
+        (0, 0.0, 0.52, 6000),
+    ],
+    "serene": [
+        (179, 0.1, 0.91, 3500),
+        (215, 0.85, 0.98, 3500),
+        (205, 0.44, 0.37, 3500),
+        (94, 0.63, 0.25, 3500),
+        (100, 0.26, 0.42, 3500),
+        (132, 0.46, 0.88, 3500),
+        (211, 0.73, 0.97, 3500),
+    ],
+    "soothing": [
+        (336, 0.18, 0.67, 3500),
+        (335, 0.5, 0.67, 3500),
+        (0, 0.0, 1.0, 8000),
+        (302, 0.69, 1.0, 3500),
+        (330, 0.45, 0.58, 3500),
+    ],
+    "sports": [(59, 0.81, 0.96, 3500), (120, 1.0, 0.96, 3500), (120, 0.74, 1.0, 3500)],
+    "spring": [
+        (184.0, 1.0, 0.5, 3500),
+        (299.0, 1.0, 0.5, 3500),
+        (49.0, 1.0, 0.5, 3500),
+        (198.0, 1.0, 0.5, 3500),
+    ],
+    "tranquil": [
+        (0, 0.0, 0.0, 8500),
+        (205, 0.74, 0.96, 3500),
+        (203, 0.94, 0.96, 3500),
+        (241, 0.99, 1.0, 3500),
+        (37, 0.75, 0.99, 3500),
+        (43, 0.83, 0.53, 3500),
+    ],
+    "warming": [
+        (4, 1.0, 0.76, 3500),
+        (42, 0.36, 0.96, 3500),
+        (355, 0.81, 0.86, 3500),
+        (44, 0.44, 0.65, 3500),
+        (51, 0.85, 0.59, 3500),
+        (0, 0.0, 0.3, 3500),
+    ],
+}
+
```

Comment:
Can you move these into aiolifx ?

Reply:
I don't think it would fit there: this has nothing to do with the actual workings of the protocol.

Reply:
It might suit `aiolifx_effects` though: I'll give that a whirl during the week.

Reply:
Perfect 👍